Line/iOS/LINELoginDemo/LINELoginDemo/HUDView.m

```diff
@@ -0,0 +1,61 @@
+//
+//  HUDView.m
+//  LINELoginDemo
+//
+//  Created by Khanh LeViet on 10/7/16.
+//  Copyright (c) Google Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+#import "HUDView.h"
+
+
+static const CGFloat kLoadingViewSize = 80;
+static const CGFloat kLoadingViewCornerRadius = 6.0;
+
+@implementation HUDView
+
++ (instancetype)generateToView:(UIView *)view {
+    HUDView *hud = [[HUDView alloc] init];
+    
+    UIView *loadingView = [self getLoadingView];
+    [hud addSubview:loadingView];
+    hud.frame = view.bounds;
+    loadingView.center = hud.center;
+    [view addSubview:hud];
+    [view bringSubviewToFront:hud];
+    
+    return hud;
+}
+
+- (void)dismiss {
+    [self removeFromSuperview];
+}
+
++ (UIView *)getLoadingView {
```

Comment:
Fixed in this [commit](https://github.com/firebase/custom-auth-samples/pull/1/commits/cfb5c9d622d615a16b4c27754be32543065571cc) :)